### PR TITLE
fix: release-expiration cronjob incorrect directory

### PR DESCRIPTION
**Pull request by theflockers**
This PR changes the output temp directory used by the cronjob `remove-expired-releases`

### CODE REVIEW

File: components/release/base/cronjobs/remove-expired-releases.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ spec:
                   NOW=$(date +%s)
                   kubectl get release --all-namespaces \
                   --sort-by=.status.expirationTime \
-                  --template '{{range .items}}{{.metadata.name}}{{"\t"}}{{.metadata.namespace}}{{"\t"}}{{.status.expirationTime}}{{"\n"}}{{end}}' > /tmp/kubectl-out
+                  --template '{{range .items}}{{.metadata.name}}{{"\t"}}{{.metadata.namespace}}{{"\t"}}{{.status.expirationTime}}{{"\n"}}{{end}}' > /var/tmp/kubectl-out
```
Comment (mmalina) [inline components/release/base/cronjobs/remove-expired-releases.yaml:33]: You will also need to change the other place that reads the file. But also, I'd suggest using `mktemp`.
```
OUTPUT=$(mktemp)
.. > $OUTPUT
...
awk ... $OUTPUT
```

Comment (mmalina) [reply components/release/base/cronjobs/remove-expired-releases.yaml]: So you're saying plain `mktemp` won't work here, right? Fine in that case.

**Review by mmalina — APPROVED** [review]

File: components/release/base/cronjobs/remove-expired-releases.yaml (hunk 1 above)
Comment (theflockers) [reply components/release/base/cronjobs/remove-expired-releases.yaml]: exactly. thanks!

**Review by happybhati — APPROVED** [review]
/lgtm

**Pull request merged**